spec/biker_spec.rb

```diff
@@ -0,0 +1,70 @@
+require './lib/ride'
+require './lib/biker'
+
+include Ride
+
+class Biker
+  attr_reader :name,
+              :acceptable_terrain,
+              :max_distance
+
+  def initialize(name, acceptable_terrain)
+    @name = name
+    @acceptable_terrain = acceptable_terrain
+    @max_distance = max_distance 
+  end
+
+  it 'has a name' do
+    biker = Biker1.new("Kenny")
+    name = biker1.name
+    expect(name).to eq ("Kenny")
+  end  
+
+  it 'has an acceptable_terrain' do
+    acceptable_terrain = Acceptable_terrain.new(["gravel", "hills"])
+    acceptable_terrain = biker1.acceptable_terrain
+    expect(acceptable_terrain).to eq (["gravel", "hills"])
+  end  
+
+  it 'has a max distance' do
+    max_distance = Max_distance.new(30)
```

Comment:
Be careful about what is an attribute and what is a class. max_distance is an attribute on the BIker class, but not a class itself. So here, you could create a Biker object and then call `.max_distance` on that biker attribute